arroyo/backends/kafka/consumer.py

```diff
@@ -161,6 +161,7 @@ def __init__(
         )
 
         configuration = dict(configuration)
+        self.__assignment_strategy = configuration.get("partition.assignment.strategy")
```

Comment:
sorry i said the wrong thing earlier, this should be `group.protocol`

Reply:
after discussing offline i think we can support KIP-848 (`group.protocol`) as well as cooperative-sticky rebalancing. they're the same as far as rdkafka API is concerned. i just can't get it to work right now and might scope it out of this PR if it takes too much time.